README.md

```diff
@@ -535,6 +535,10 @@ Something is missing?  Create a pull request!
         [github](https://github.com/incapptic)
         ,
         [jobs](https://www.incapptic.com/company-careers/)
+    * [Infarm - Indoor Urban Farming GmbH](https://infarm.com/)
+        [github](https://github.com/infarm/)
```

Comment:
- I tried to find use of Django at…

  - https://github.com/infarm/?language=python and
  - https://www.wappalyzer.com/lookup/infarm.com/

  …but didn't find any evidence myself.
  Please add a link for proof with use of Django visible to the public internet as all the other companies in this list have. E.g. a link to job ad mentioning Django (outside of "Laravel or Django or RoR") or a link to Django admin or `staticfiles.json` would do.
- An emdesh is missing between the company name and the link list that follows.
- https://www.infarm.com/en/imprint reads "InFarm – Indoor Urban Farming GmbH" with capital "F". is captial "F" or lower "f" correct? 

Thanks! :pray:

Reply:
thanks for your feedback. I think I addressed the points.

Reply:
Excellent — thank you! :pray: